tests/TestRvsdgs.hpp

```diff
@@ -1807,4 +1812,146 @@ class LinkedListTest final : public RvsdgTest
   jlm::rvsdg::node * Alloca_;
 };
 
+/** \brief RVSDG module with one of each memory node type.
+ *
+ * The class sets up an RVSDG module corresponding to the code:
+ *
+ * \code{.c}
+ *   static int global = 1;
```

Comment:
This is static and nowhere used in the code, and therefore the delta is not exported. This means DNE will just remove it if it is ever used on it. I am not sure that is what you want!?

Reply:
This TestRVSDG subclass doesn't really represent a useful program, so I agree that any DNE pass applied to it would remove unused things (same applies to the import and alloca). I can try to make the program a bit more realistic, to perhaps make the test class more generally useful.

Reply:
It is fine as it is, I just wanted to point it out in  case it was not a wanted side-effect.

Reply:
Btw, did you make any changes after my last round of comments? If so, did you forget to push them as I see no differences?